apps/showcase/src/components/showcase/rules-engine/rules-engine-pres.config.ts

```diff
@@ -39,4 +39,4 @@ export const RULES_ENGINE_PRES_DEFAULT_CONFIG: RulesEnginePresConfig = {
   shouldProposeRoundTrip: false
 };
 
-export const RULES_ENGINE_PRES_CONFIG_ID = computeConfigurationName('RulesEnginePresConfig', 'showcase');
+export const RULES_ENGINE_PRES_CONFIG_ID = computeComponentIdentifier('RulesEnginePresConfig', 'showcase');
```

Comment:
Are we sure about this change ?
as here we give the configuration name as 1st param and not the component name
we have to be carefully with that has sometimes we need the config id and sometimes the component id